app/renderer/modals/NewBookModal.tsx

```diff
@@ -20,6 +20,7 @@ const NewBookModal = (props: ModalProps) => {
   const formRef = useRef<HTMLFormElement>(null);
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
I don't think the Button in the form doesn't need the `onClick` prop to be set anymore, so it should be removed to prevent the form from getting submitted twice.